### ISSUE TITLE
GW2000

### ISSUE THREAD

**Issue by Batman6997** [spatialdude/homebridge-ecowitt#21]
I've been able to get this plugin to collect data from my GW2000, however it doesn't create any accessories in Homekit.

Tried the fork by @evansunleyjames and it seems to exclude my GW2000B version.

Is there any way to fix this?

**Comment by Batman6997** [comment]
Appears that there is no accessory for my weather station. Managed to get the data populated by manually editing the file for WH65, changing it to fit my WS90. Wish I knew how to this properly so everyone can have access. 
Now need to figure out how to get the tiles in my home app.

**Comment by robsorbo** [comment]
Any success getting tiles into the Home app?

**Comment by Batman6997** [comment]
No, it appears that HomeKit doesn’t allow for weather related tiles

**Comment by evansunleyjames** [comment]
Hey, sorry I didn't see the mention here @Batman6997. I haven't had the time to put into this, but feel free to make a PR on my fork if you'd like - maybe all it needs is a little smarter string matching. My experience with using this with HomeKit, like yours, isn't great - I do have the temp/humidity going in there but I couldn't make anything useful out of the other sensors due to the lack of sensor types in the SDK - having rainfall as a leak sensor, for example, isn't great.

**Comment by Batman6997** [comment]
It’s been a so long since I did this, but I know that I manually edited a few files right from terminal. I believe  I changed the GW2000C to GW2000 and tricked it into recognizing my WS90 as a WH65.
Tried to create my own fork, but have no idea how to do this or to properly compile into a repo. Let alone perform a PR

**Comment by rhockenbury** [comment]
If it helps at all, I have a new Ecowitt plugin available (v2) that adds support for the GW2000, WS90, and WH65.  Check out [Homebridge Ecowitt Weather Sensors](https://github.com/rhockenbury/homebridge-ecowitt-weather-sensors) which can be installed through the Homebridge UI by searching for `homebridge-ecowitt-weather-sensors`.  Would appreciate any feedback if you decide to give this new plugin a try, and [let me know if you hit any issues](https://github.com/rhockenbury/homebridge-ecowitt-weather-sensors/issues/new).
